language/e2e-tests/src/common_transactions.rs

```diff
@@ -25,7 +25,7 @@ pub fn add_validator_txn(
         vec![],
         args,
         seq_num,
-        gas_costs::TXN_RESERVED,
+        gas_costs::TXN_RESERVED * 2,
```

Comment:
?

Reply:
forget to mention, this is because the contract runs out of gas likely due to we copy the validator set instead of mutating in-place now

Reply:
This seems rather sad... is there a way to fix this? or increase TXN_RESERVED to be 2x?

Reply:
>  increase TXN_RESERVED to be 2x

This breaks a bunch of tests in a way I don't understand, seems we assert some gas amount in tests.